addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/servlet/CometVisuServlet.java

```diff
@@ -768,6 +774,43 @@ else if (!contentType.startsWith("image")) {
     }
 
     /**
+     * Save config file send by editor
+     *
+     * @param request
+     * @param response
+     * @throws ServletException
+     * @throws IOException
+     */
+    private final void saveConfig(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String fileName = request.getParameter("config");
+        File file = new File(userFileFolder, URLDecoder.decode(fileName, "UTF-8"));
```

Comment:
Should we check for a fileName that begins with e.g. "../"?
Of is this already checked somewhere?
A common use case for file paths in URL parameters is using a lot of "../../../../../../../" (etc.) so we know we are on the root directory and then go again below and doing so read / write files.